packages/@o3r/rules-engine/src/services/rules-engine.service.ts

```diff
@@ -1,10 +1,12 @@
 import { Inject, Injectable, OnDestroy, Optional } from '@angular/core';
 import { select, Store } from '@ngrx/store';
 import { TranslateService } from '@ngx-translate/core';
-import { deletePlaceholderTemplateEntity, PlaceholderTemplateReply, PlaceholderTemplateStore, selectPlaceholderTemplateEntities, setPlaceholderTemplateEntityFromUrl } from '@o3r/components';
+import type { PlaceholderTemplateStore } from '@o3r/components';
+import { deletePlaceholderTemplateEntity, PlaceholderTemplateReply, selectPlaceholderTemplateEntities, setPlaceholderTemplateEntityFromUrl } from '@o3r/components';
 import { computeConfigurationName, ConfigurationStore, PropertyOverride, setConfigOverride } from '@o3r/configuration';
 import { AssetPathOverrideStore, DynamicContentService, setAssetPathOverride } from '@o3r/dynamic-content';
-import { LocalizationOverrideStore, setLocalizationOverride } from '@o3r/localization';
+import type { LocalizationOverrideStore } from '@o3r/localization';
+import { setLocalizationOverride } from '@o3r/localization';
```

Comment:
Not from this PR but the last type from the Store generic in the constructor should be `ConfigOverrideStore` instead of `ConfigStore`